ts.eclipse.ide.angular2.cli/src/ts/eclipse/ide/angular2/internal/cli/launch/AngularCLILaunchConfigurationDelegate.java

```diff
@@ -61,24 +61,30 @@ public void launch(ILaunchConfiguration configuration, String arg1, ILaunch laun
 			return;
 		}
 
+		IPath processWorkingDir = workingDir;

```

Comment:
I just pulled from your Repo before pushing to my fork, strange that my fork isn't synced to yours...  
I'll try it again tonight!